DOCS.md

```diff
@@ -53,7 +53,7 @@ See [this article](https://www.gitpod.io/blog/open-vsx/) for more information on
 
 ### <a id="howto-vscode-marketplace"></a>How to use the VS Code Marketplace
 
-You can switch and use the VS Code marketplace by using the following solutions. However, note that [it is not clear whether this is legal](https://github.com/microsoft/vscode/issues/31168).
+You can switch and use the VS Code marketplace by using the following solutions. However, note that [ this is not legal](https://github.com/microsoft/vscode/issues/31168).
```

Comment:
It is unclear to me.

Reply:
The additional information you added around OpenVSX and how to get extensions published at open-vsx.org is excellent.

This is why I always check the "Allow edits by maintainers" option for my pull requests. Thank you.

Reply:
> The additional information you added around OpenVSX and how to get extensions published at open-vsx.org is excellent.
> 
> This is why I always check the "Allow edits by maintainers" option for my pull requests. Thank you.

I was not maintainer, I had access to submit tho :)